Tools/clinic/clinic.py

```diff
@@ -5832,6 +5832,8 @@ def create_cli() -> argparse.ArgumentParser:
                          help="walk --srcdir to run over all relevant files")
     cmdline.add_argument("--srcdir", type=str, default=os.curdir,
                          help="the directory tree to walk in --make mode")
+    cmdline.add_argument("--exclude", type=str, metavar="EXCLUDES",
+                         help="a comma-separated list of files to exclude in --make mode")
```

Comment:
Instead of a comma-separated list I would allow to specify the option multiple times. Use `action='append'`.

Reply:
Yes, that makes for a better user experience.

Reply:
Resolved.